src/api/systems.tsx

```diff
@@ -44,6 +45,15 @@ const fetchSystems = async (parent_id?: string): Promise<System[]> => {
   });
 };
 
+export const useSystemIds = (ids: string[]): UseQueryResult<System>[] => {
```

Comment:
I think this should have a unit test like `useCatalogueItemIds`/`useManufacturerIds`